examples/dwarfdump.rs

```diff
@@ -497,7 +497,7 @@ fn main() {
                 process::exit(1);
             }
         };
-        let mmap_ref = (*arena_mmap.alloc(mmap)).borrow();
+        let mmap_ref = &*arena_mmap.alloc(mmap);
```

Comment:
These changes are good, but it seems like this is a false positive for the rust compiler warning?

Reply:
Yeah, I think you're right. I've changed this a bit to simplify.

Reply:
Ah nice, even better.